java/plugins/html-elements-plugin/src/test/java/setting/JettySettings.java

```diff
@@ -0,0 +1,63 @@
+package setting;
+
+import cucumber.api.java.Before;
+import org.eclipse.jetty.jmx.MBeanContainer;
+import org.eclipse.jetty.server.Server;
+import org.eclipse.jetty.webapp.WebAppContext;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import ru.sbtqa.tag.pagefactory.PageFactory;
+
+import java.io.File;
+import java.lang.management.ManagementFactory;
+
+import static java.lang.Runtime.getRuntime;
+
+/**
+ * Created by Liza on 07.06.17.
+ */
+public class JettySettings {
+
+    private static final Logger LOG = LoggerFactory.getLogger(JettySettings.class);
+    
+    public static final String WAR_PATH = "src/test/resources/test-web-app.war";
+    public static final int PORT = 8181;
+    
+    private static boolean dunit = true;
+    
+    private Server server;
+    
+    @Before(order = 999)
+    public void startJetty() throws Exception {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00112)